### DISCUSSION TITLE
Compiling errors

### DISCUSSION THREAD

**Discussion by aminghaith24** [svenreiche/Genesis-1.3-Version4#110] [Q&A]
Hello,
I am encountering some errors in the compiling during the make command.
error: 'FFTObj' does not name a type 
And if I comment this FFTObj::FFTObj(int ngrid) alongside the rest , I get the new errors:

void DiagField::cleanup_FFT_resources(void)
error: ‘fftw_plan’ has not been declared

Thanks in advance

**Comment by ZeugAusHH** [comment]
Hello,

a few questions regarding the compilation issue you are reporting:
Which GENESIS source code version are you using (which git commit id)?
Which compiler and compiler version are you using? Which is the version of your FFTW?
Finally, could you please post the output of your cmake run?

Thank you
Christoph

**Comment by aminghaith24** [comment]
Hello

Had to update FFTW, now genesis's all compiled and installed,  thank you so much for your quick response.

Best regards.

**Comment by svenreiche** [comment]
This was a bug where not all elements related to FFTW weren't excluded during compilation. In the current dev branch it is fixed.  In the future FFTW might become mandatory then the cmake with not produce any longer a makefile.
